src/driftpy/drift_user.py

```diff
@@ -226,9 +349,9 @@ def get_total_collateral(
     def get_free_collateral(
         self, margin_category: MarginCategory = MarginCategory.INITIAL
     ):
-        total_collateral = self.get_total_collateral(margin_category)
-        init_margin_req = self.get_margin_requirement(margin_category)
-        free_collateral = total_collateral - init_margin_req
+        total_collateral = self.get_total_collateral(margin_category, True)
```

Comment:
should only use strict for init and should use it for total collateral and margin requirement?